org.photoqt.PreviewQt.yml

```diff
@@ -0,0 +1,311 @@
+app-id: org.photoqt.PreviewQt
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+command: previewqt
+finish-args:
+    - --share=ipc
+    - --socket=wayland
+    - --socket=fallback-x11
+    - --filesystem=host
```

Comment:
host is a very broad filesystem permission

Reply:
I understand. The hope is that this app (eventually) will be able to preview as many files as possible, no matter where they live.